### PR TITLE
Room updates

### PR DESCRIPTION
**Pull request by ricco386**
Improoved admin for location and room
Minor model update

### CODE REVIEW

File: konfera/models/room.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 class Room(models.Model):
     title = models.CharField(max_length=128)
+    room = models.ForeignKey('Location', related_name='room_locations')
```
Comment (ZoltanOnody) [inline konfera/models/room.py]: `location = ...` not `room`

I'd change the related name to `rooms` it is in my opinion more descriptive when you use it.
```
location = Location.objects.get(...)
location.rooms.all()  # seems better to me
location.room_locations.all()
```

Comment (ricco386) [inline konfera/models/room.py]: :+1: thanks for spotting this. Updated.

Comment (ZoltanOnody) [inline konfera/models/room.py]: I would change the `room =` to `location =`.

Comment (ricco386) [inline konfera/models/room.py]: updated

**Pull request merged**